overrides/build-systems.json

```diff
@@ -1426,6 +1426,9 @@
   "asyncinotify": [
     "flit-core"
   ],
+  "asyncio": [
```

Comment:
This has been in the Python standard library since 3.4 and nix dropped support for that version of Python long before the last minimum nixpkgs requirement in poetry2nix.

How is it possible that you're hitting a requirement to build this library as a package?